### PR TITLE
lca cnf: use IBGU in e2e-upgrade-test

### PR DESCRIPTION
**Pull request by mcornea**
Depends-On: https://github.com/openshift-kni/eco-goinfra/pull/708

### CODE REVIEW

File: tests/lca/imagebasedupgrade/cnf/upgrade-talm/internal/tsparams/const.go [hunk 1]
```diff
@@ -15,6 +15,15 @@ const (
 	// IbuCguNamespace is the namespace where IBU CGUs created on target hub.
 	IbuCguNamespace = "default"
 
+	// IbguNamespace is the namespace where IBU CGUs created on target hub.
```
Comment (achuzhoy) [inline tests/lca/imagebasedupgrade/cnf/upgrade-talm/internal/tsparams/const.go:18]: Same comment as for IbuCguNamespace above.

Comment (mcornea) [reply tests/lca/imagebasedupgrade/cnf/upgrade-talm/internal/tsparams/const.go]: tnx, updated.

File: tests/lca/imagebasedupgrade/cnf/upgrade-talm/tests/e2e-upgrade-test.go [hunk 2]
```diff
@@ -51,68 +50,32 @@ var _ = Describe(
...
-					tsparams.IbuCguNamespace)
-				Expect(err).ToNot(HaveOccurred(), "Failed to delete rollback cgu on target hub cluster")
-			})
-
-			By("Deleting finalize cgu created on target hub cluster", func() {
-				err := cnfhelper.DeleteIbuTestCguOnTargetHub(cnfinittools.TargetHubAPIClient, tsparams.FinalizeCguName,
-					tsparams.IbuCguNamespace)
-				Expect(err).ToNot(HaveOccurred(), "Failed to delete finalize cgu on target hub cluster")
+			By("Deleting upgrade ibgu created on target hub cluster", func() {
+				upgradeIbguBuilder := ibgu.NewIbguBuilder(
```
Comment (josclark42) [inline tests/lca/imagebasedupgrade/cnf/upgrade-talm/tests/e2e-upgrade-test.go:54]: In AfterAll you build a new IBGU upgradeIbguBuilder in order to delete it.  I understand why.

[In my PR I declare ](https://github.com/openshift-kni/eco-gotests/pull/222/files#diff-dea03563f03fb7c1e3c01004644f8e797e837b4b4bf16073963091cfef0997a5R19) "var newIbguBuilder *ibgu.IbguBuilder"  in Describe so in the AfterAll I don't need to build it again, I can just newIbguBuilder.DeleteAndWait

Both work.. is one way better than the other?

Comment (mcornea) [reply tests/lca/imagebasedupgrade/cnf/upgrade-talm/tests/e2e-upgrade-test.go]: thanks Josh, I've updated the test to use the same approach as in your test for consistency.

**Review by josclark42 — APPROVED** [review]

**Pull request merged**